dockerfiles/entity-providers/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:18-alpine
+MAINTAINER Roadie
+WORKDIR /usr/src/app
+COPY package.json .
+RUN npm install && npm install typescript -g
+COPY . .
+RUN yarn build
+CMD ["node", "./bin/kubernetes-entity-provider"]
```

Comment:
need the ability to configure the entity provider entity mappings and the broker client server url and token.